app/assets/stylesheets/components/_feed.scss

```diff
@@ -0,0 +1,102 @@
+.feed{
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  gap: 1rem;
+
+  .user{
+    background-color: $feed_card;
+    padding: 1rem;
+    display: grid;
+
+    &__img{
+      width: 100%;
+    }
+
+    &__name{
+      margin: 0;
+      margin-top: 1rem;
+      text-align: center;
+    }
```

Comment:
Please, in each of this file blocks add a space between your name class and  brackets